---
Class: University
Status: Done
Priority:
  - High
Week: 1
Lecture:
  - ðŸŸ©
Flashcards:
  - ðŸŸ¥
tags:
  - "#SCIE4401"
---
> Error generating daily quote
- [x] ðŸ“… 2024-07-22 Rank Papers for presentation [[SCIE4401 Presentation]] âœ… 2024-08-01
- [x] ðŸ›« 2024-07-30 Release of assignment âœ… 2024-08-02
---
# Timetable
![[Timetable SCIE4001 Lectures 2024 S2 v2.pdf]]

---
# Assessment Information
![[Assesment overview.png]]
<!--âš ï¸Imgur upload failed, check dev console-->
![](https://i.imgur.com/NnSG8Id.png)


---
# Notes for SCIE4401 Introduction
![[genotype to phenotype L1- 2023-sem2.pdf#page=5&rect=24,196,321,509&color=yellow|genotype to phenotype L1- 2023-sem2, p.5]]simple highschool level, start here
> [!PDF|yellow] [[genotype to phenotype L1- 2023-sem2.pdf#page=6&selection=38,0,40,27&color=yellow|genotype to phenotype L1- 2023-sem2, p.6]]
> > There is no dedicated textbook, but if you do not have a strong genetic background please refer to:
> 
> if struggling get a text book and read

> [!PDF|yellow] [[genotype to phenotype L1- 2023-sem2.pdf#page=7&selection=0,0,81,1&color=yellow|genotype to phenotype L1- 2023-sem2, p.7]]
> > What is functional genomicsâ€¦â€¦â€¦Functional genomics uses genomic data to study gene and protein expression and function on a global scale (genome-wide or system-wide), focusing on gene transcription, translation and protein-protein interactions, and often involving high-throughput methods.
> 
> exceptions to this rule, e.g. non codin rna miRNA etc etc


> [!PDF|yellow] [[genotype to phenotype L1- 2023-sem2.pdf#page=9&selection=14,0,14,13&color=yellow|genotype to phenotype L1- 2023-sem2, p.9]]
> > Phylogenetics
> 
> how related are genes, species diverge (when a gene has an important function it is maintained how is it maintained in a population)

> [!PDF|yellow] [[genotype to phenotype L1- 2023-sem2.pdf#page=9&selection=18,0,24,12&color=yellow|genotype to phenotype L1- 2023-sem2, p.9]]
> > Create variants and analyse the phenotypic Consequences
> 
> bioinformatic analysis = finding what gene to change and then seeing the results of changing that gene

> [!PDF|yellow] [[genotype to phenotype L1- 2023-sem2.pdf#page=10&selection=2,0,4,8&color=yellow|genotype to phenotype L1- 2023-sem2, p.10]]
> > Functional genomics
> 
> can be biased to theory, may work in a closed system but not inn a real system. Cells can adjust to mutations.

> [!PDF|yellow] [[genotype to phenotype L1- 2023-sem2.pdf#page=11&selection=0,0,0,35&color=yellow|genotype to phenotype L1- 2023-sem2, p.11]]
> > Why do we need functional genomics?
> 
> we still dont kow what anything does

> [!PDF|yellow] [[genotype to phenotype L1- 2023-sem2.pdf#page=11&selection=6,0,8,17&color=yellow|genotype to phenotype L1- 2023-sem2, p.11]]
> > % of genes with inferred function
> 
> inferred function = genes annotated based on homology

> [!PDF|yellow] [[genotype to phenotype L1- 2023-sem2.pdf#page=11&selection=48,0,49,0&color=yellow|genotype to phenotype L1- 2023-sem2, p.11]]
> > Arabidopsis
> 
> although sequenced we still do not know what many genes do

> [!PDF|yellow] [[genotype to phenotype L1- 2023-sem2.pdf#page=12&selection=2,0,2,20&color=yellow|genotype to phenotype L1- 2023-sem2, p.12]]
> > Arabidopsis thaliana
> 
> weed that grows in cracks in Europe

![[genotype to phenotype L1- 2023-sem2.pdf#page=12&rect=252,217,463,385&color=yellow|genotype to phenotype L1- 2023-sem2, p.12]]fast repro and growth rate from seed to plant, good for experimenting - makes it a good model organism

> [!PDF|yellow] [[genotype to phenotype L1- 2023-sem2.pdf#page=12&selection=20,0,20,17&color=yellow|genotype to phenotype L1- 2023-sem2, p.12]]
> > Small genome size
> 
> diploid - 2 copies of gene so it is small (like humans are diploid)

> [!PDF|yellow] [[genotype to phenotype L1- 2023-sem2.pdf#page=12&selection=34,0,34,18&color=yellow|genotype to phenotype L1- 2023-sem2, p.12]]
> > Easily manipulated
> 
> refers to transgenics in lab


![[genotype to phenotype L1- 2023-sem2.pdf#page=14&rect=85,420,290,460&color=yellow|genotype to phenotype L1- 2023-sem2, p.14]]phenotype first then go back and find the genetic code

> [!PDF|yellow] [[genotype to phenotype L1- 2023-sem2.pdf#page=17&selection=2,40,2,53&color=yellow|genotype to phenotype L1- 2023-sem2, p.17]]
> > cell cultures
> 
> easy to treat them for chemicals

![[genotype to phenotype L1- 2023-sem2.pdf#page=18&rect=87,188,316,402&color=yellow|genotype to phenotype L1- 2023-sem2, p.18]]multiple copies  of one gene and some genes duplicated and slightly duplicated / evolved, makes it difficult to work with


> [!PDF|yellow] [[genotype to phenotype L1- 2023-sem2.pdf#page=19&selection=10,35,10,36&color=yellow|genotype to phenotype L1- 2023-sem2, p.19]]
> > T
> 
> Arabidopsis Taliana (?)

> [!PDF|yellow] [[genotype to phenotype L1- 2023-sem2.pdf#page=19&selection=10,34,10,43&color=yellow|genotype to phenotype L1- 2023-sem2, p.19]]
> > AT5g15960
> 
> on chromosome 5 and at that position


> [!PDF|yellow] [[genotype to phenotype L1- 2023-sem2.pdf#page=20&selection=0,0,2,18&color=yellow|genotype to phenotype L1- 2023-sem2, p.20]]
> > The sequence of the first plant genome was completed and published at the end of 2000
> 
> australian variant is Arabidopsis collumbia - different variants and species may have resistance to certain environmental factors

![[genotype to phenotype L1- 2023-sem2.pdf#page=21&rect=10,3,362,502&color=yellow|genotype to phenotype L1- 2023-sem2, p.21]]insert random TDNA, segregated populations with different TDNA insertion, 1 TDNA insertion to 1 gene

> [!PDF|yellow] [[genotype to phenotype L1- 2023-sem2.pdf#page=22&selection=6,0,6,13&color=yellow|genotype to phenotype L1- 2023-sem2, p.22]]
> > Agrobacterium
> 
> soil bacteria which infects plants by transferring DNA into host plant genome.

> [!PDF|yellow] [[genotype to phenotype L1- 2023-sem2.pdf#page=22&selection=30,0,32,3&color=yellow|genotype to phenotype L1- 2023-sem2, p.22]]
> > T-DNA
> 
> Transfer DNA

![[genotype to phenotype L1- 2023-sem2.pdf#page=26&rect=74,64,393,268&color=yellow|genotype to phenotype L1- 2023-sem2, p.26]]primer outside of gene and primer inside of gene

![[genotype to phenotype L1- 2023-sem2.pdf#page=27&rect=15,32,350,175&color=yellow|genotype to phenotype L1- 2023-sem2, p.27]]2 different lines for same gene. Done because they are genetically independent. Be sure that thise gene is responsible for the phenotype. COnfirmation basically

> [!PDF|yellow] [[genotype to phenotype L1- 2023-sem2.pdf#page=29&selection=30,0,30,32&color=yellow|genotype to phenotype L1- 2023-sem2, p.29]]
> > How do you screen mutant lines ?
> 
> simplification, S1 would show the gene because primers capture whole gene, S2 shows T-DNA insertion because primer is on T-DNA insertion point


> [!PDF|yellow] [[genotype to phenotype L1- 2023-sem2.pdf#page=30&selection=0,0,2,8&color=yellow|genotype to phenotype L1- 2023-sem2, p.30]]
> > Genotyping: What happens if you cannot find a homozygous knock-out line
> 
> e.g. a gene essential for life

![[genotype to phenotype L1- 2023-sem2.pdf#page=30&rect=39,247,516,424&color=yellow|genotype to phenotype L1- 2023-sem2, p.30]]deleting protein essential to life, 3 to 1 ratio of homozygous based on mendels laws (white seeds), the white seeds will not sprout


> [!PDF|yellow] [[genotype to phenotype L1- 2023-sem2.pdf#page=31&selection=288,0,288,33&color=yellow|genotype to phenotype L1- 2023-sem2, p.31]]
> > How do you confirm mutant lines ?
> 
> via quantitative reverse PCR, looks only at one gene, cheaper than RNA sequencing

![[SCIE4001 - 2022 Transcriptomics lecture.pdf#page=6&rect=250,14,937,444&color=yellow|SCIE4001 - 2022 Transcriptomics lecture, p.6]]alot of the sequence is repetitive

![[SCIE4001 - 2022 Transcriptomics lecture.pdf#page=14&rect=521,56,922,259&color=yellow|SCIE4001 - 2022 Transcriptomics lecture, p.14]]
can lie on introns or on exons (antisense strand)

> [!PDF|yellow] [[SCIE4001 - 2022 Transcriptomics lecture.pdf#page=19&selection=8,0,9,0&color=yellow|SCIE4001 - 2022 Transcriptomics lecture, p.19]]
> > ESTs
> 
> expressed sequenced tags, polyadenlyated rna, short read of transcript that gave evidence that it was being expressed - low throughput, large endeavour to do over many genes

> [!PDF|yellow] [[SCIE4001 - 2022 Transcriptomics lecture.pdf#page=19&selection=26,0,27,1&color=yellow|SCIE4001 - 2022 Transcriptomics lecture, p.19]]
> > Microarray 
> 
> printed surface where dna spots printed on slide which allow to detect hybridized sequences

> [!PDF|yellow] [[SCIE4001 - 2022 Transcriptomics lecture.pdf#page=19&selection=34,0,35,0&color=yellow|SCIE4001 - 2022 Transcriptomics lecture, p.19]]
> > RNA-seq
> 
> high throughput

![[SCIE4001 - 2022 Transcriptomics lecture.pdf#page=24&rect=14,17,1013,756&color=yellow|SCIE4001 - 2022 Transcriptomics lecture, p.24]]needs additional research

> [!PDF|yellow] [[SCIE4001 - 2022 Transcriptomics lecture.pdf#page=25&selection=2,0,2,55&color=yellow|SCIE4001 - 2022 Transcriptomics lecture, p.25]]
> > RNA-seq: massively parallel sequencing of RNA fragments
> 
> largest method is sequencing by synthesis, turn RNA into DNA which makes copies of DNA and sequences copies

> [!PDF|yellow] [[SCIE4001 - 2022 Transcriptomics lecture.pdf#page=33&selection=8,0,10,6&color=yellow|SCIE4001 - 2022 Transcriptomics lecture, p.33]]
> > Bromodeoxyuridine (BrdU)
> 
> analog of thymodine


![[SCIE4001 - 2022 Transcriptomics lecture.pdf#page=34&rect=98,320,404,675&color=yellow|SCIE4001 - 2022 Transcriptomics lecture, p.34]]eliminates RNA except for the one in the ribosome



![[SCIE4001 - 2022 Transcriptomics lecture.pdf#page=38&rect=517,397,565,438&color=yellow|SCIE4001 - 2022 Transcriptomics lecture, p.38]]poly t hybridizes to poly A sequences

> [!PDF|yellow] [[SCIE4001 - 2022 Transcriptomics lecture.pdf#page=38&selection=27,19,29,8&color=yellow|SCIE4001 - 2022 Transcriptomics lecture, p.38]]
> > 10x barcodes
> 
> each barcode is different on every bead

> [!PDF|yellow] [[SCIE4001 - 2022 Transcriptomics lecture.pdf#page=43&selection=0,0,0,34&color=yellow|SCIE4001 - 2022 Transcriptomics lecture, p.43]]
> > Spatially resolved transcriptomics
> 
> simmilar to microarray, lots of spots on surface with oligoDT which pairs to an mRNA



---
# Compressed Notes
## Lecture 1
### Basics of Functional Genomics
â€¢ Describe various targeted genetic approaches to determine the function of a protein. 
â€¢ Describe how plant phenotypes can be used to unravel a proteins function. 
â€¢ Understand the current research themes and approaches used in reverse genetics and forward genetic screens in the model Arabidopsis thaliana 
â€¢ Critically evaluate the discussed techniques available for researchers today.

***What is Functional genomics?*** 
Using genomic data to study gene and protein expression and function on a global scale (genome-wide or system-wide), focusing on gene transcription, translation and protein-protein interactions, and often involving high-throughput methods. Essentially, the study of how genes (and intergenic regions) contribute to different biological processes.

***What is the aim of functional genomics?***
To understand the complex relationship between genotype and phenotype

Investigation of gene function is split into reverse genetics and forward genetics.
Reverse genetics:
Forward genetics: 


---

# Flashcards for SCIE4401 Introduction



---
# References for SCIE4401 Introduction
![[1. Introduction_SCIE4001_2024_S2.pdf]]![[genotype to phenotype L1- 2023-sem2.pdf]]![[GENOTYPE TO PHENOTYPE L2-2.pdf]]
![[SCIE4001 - 2022 Transcriptomics lecture.pdf]]

![[2024-05Mar_SCI4001(1).pdf]]